src/guides/getting-started.md

```diff
@@ -0,0 +1,597 @@
+---
+title: "Getting Started"
+lang: en-US
+css: ../assets/stylesheets/application.css
+include-after: |
+    <script src="../assets/javascripts/application.js"></script>
+---
+
+::: demo
+::: content-wrapper
+::: guide-wrapper
+
+For this guide, we're going to walk through some simple examples for each of the pieces of CRUD,
+which stands for "Create Read Update Delete". Each step in this guide will build on the previous,
+and is meant to be followed along.
+
+**This guide assumes that you're using PostgreSQL.** Before we start,
+make sure you have PostgreSQL installed and running.
+
+<aside class = "aside aside--note">
+<header class = "aside__header">A note on Rust versions:</header>
+
+::: aside__text
+Diesel requires Rust 1.31 or later. If you're following along with this guide,
+make sure you're using at least that version of Rust by running `rustup update stable`.
+
+:::
+</aside>
+
+## Install `diesel_cli`
```

Comment:
We seem to be doing more than just installing the Diesel CLI in this section

Reply:
@weiznich I'm not sure whether you've seen this comment

Reply:
This is content that was added as part of this PR, as there was no title in the guide before.

Reply:
Thanks for the comment. I've missed this change, as I took the md files from @JohnTitor branch 🙈 
I've introduced another headline there, to make it a bit more clear what happens where.

Reply:
Oh, seems I introduced it, thanks for catching up!